data/directory/23u5y2o.json

```diff
@@ -17,7 +17,7 @@
   "activities": {
```

Comment:
is there a way to build the JSON's file name from the person's name so that the URL looks more informative?

Reply:
or maybe it is already the case with the real data?

Reply:
yes, that is already the case with the data. :) I was just lazy with the mock data